src/core/project/qgsprojecttranslator.h

```diff
@@ -33,7 +33,7 @@ class CORE_EXPORT QgsProjectTranslator
 
     /**
      * The derived translate() translates with QTranslator and qm file the sourceText.
-     * It \returns the result string and in case there is no QTranslator loaded, the sourceText.
+     * It returns the result string and in case there is no QTranslator loaded, the sourceText.
```

Comment:
```suggestion
     * \return the result string and in case there is no QTranslator loaded, the sourceText.
```

So we save the :return: tag in sip file

Reply:
Indeed, the docs for this whole class where quite poor. I've improved them all now.